### PR TITLE
Navbar design

### PR DESCRIPTION
**Pull request by PremanandChowdhury**
This is a new PR for the Navbar Component 🙏🏼 
Also, the changes are: 
- Navbar Design
- Navbar added to every page.
- Also, img tag changes to Image Component.

Issue Ref.  #107 & #114  😄

### CODE REVIEW

File: components/NavBar/navbar.module.css [hunk 1]
```diff
@@ -1,30 +1,41 @@
...
+}
+.navBar > ul {
+  display: flex;
 }
-.navBar > li {
+.navBar li {
   list-style-type: none;
   color: #f2f2f2;
-  padding: 0 0.6em;
+  /* padding: 0 0.6em; */
```
Comment (Harshi7016) [inline components/NavBar/navbar.module.css:19]: you can remove the comments if you find it not necessary

File: components/shoplist-card/index.js [hunk 2]
```diff
@@ -26,62 +35,65 @@ export const ShopCard = ({ product, quantity, add, del, link }) => {
...
+          }
+          .shop-card-product-qnty {
+            display: flex;
+            font-size: 1.3em;
+            width: 100%;
+            justify-content: space-evenly;
+            align-items: center;
+          }
+        `}
+      </style>
```
Comment (Harshi7016) [inline components/shoplist-card/index.js:96]: Is this just indention changes?

File: pages/index.js [hunk 3]
```diff
@@ -9,7 +9,7 @@ import { Footer } from '../components/footer';
 import styles from '../styles/Home.module.css';
 import TransactionChart from '../components/transaction-chart';
 import TransactionOperationModal from '@components/transaction-operation-modal';
```
Comment (Harshi7016) [inline pages/index.js:11]: A request, can you change this to ../components/transaction-operation-modal :)

File: styles/globals.css [hunk 4]
```diff
@@ -63,6 +63,7 @@ footer a {
 
 .main-container {
   height: 100%;
+
```
Comment (Harshi7016) [inline styles/globals.css:66]: I am not sure why there is an extra space here

File: components/shoplist-card/index.js (hunk 2 above)
Comment (PremanandChowdhury) [reply components/shoplist-card/index.js:95]: yes 😅

File: pages/index.js (hunk 3 above)
Comment (PremanandChowdhury) [reply pages/index.js:11]: ok, Will be done

File: styles/globals.css (hunk 4 above)
Comment (PremanandChowdhury) [reply styles/globals.css:66]: which space are you talking about? @Harshi7016

Comment (Harshi7016) [reply styles/globals.css:66]: Extra line in globals.css , this can be  ignored

Comment (PremanandChowdhury) [reply styles/globals.css:66]: O will be removed 👍

**Comment by Kratika0907** [comment]
Prem screenshots please

**Comment by PremanandChowdhury** [comment]
> 
> 
> Prem screenshots please

**Comment by Harshi7016** [comment]
You are try loom for video too, check it out

**Comment by PremanandChowdhury** [comment]
Homepage Desktop View:
Desktop View

Homepage Mobile View:
Mobile View

Mobile View of shop page:
Shop Mobile View

@Harshi7016 Next time for sure i'll upload a video using loom.

**Review by Harshi7016 — APPROVED** [review]

**Review by Harshi7016 — APPROVED** [review]

**Review by Kratika0907 — APPROVED** [review]

**Pull request merged**